research/slim/nets/mobilenet_v1_test.py

```diff
@@ -310,7 +350,7 @@ def testModelHasExpectedNumberOfParameters(self):
       mobilenet_v1.mobilenet_v1_base(inputs)
       total_params, _ = slim.model_analyzer.analyze_vars(
           slim.get_model_variables())
-      self.assertAlmostEqual(3217920, total_params)
+      self.assertAlmostEqual(3217920L, total_params)
```

Comment:
The trailing __L__ is no longer needed in Python 2 and is a __SyntaxError__ in Python 3.

python3 -c "3217920L" # --> SyntaxError: invalid syntax

Please remove the trailing __L__ to be compatible with both Python2 and Python 3.